cloud/linode/instances_test.go

```diff
@@ -233,17 +233,6 @@ func TestMalformedProviders(t *testing.T) {
 
 	client := NewMockClient(ctrl)
 
-	t.Run("fails on malformed providerID", func(t *testing.T) {
```

Comment:
why remove? can't fix?

Reply:
there's no "malformed provider ID" anymore, its either a linode-id or a label that we can look up - since we mock the linodeGo response, I didn't feel like it was a useful test to see if a string existed in a list (that we mock)

Reply:
This is not what the test does, it doesn't test anything in the linodego response - it checks that we error out in case there's an incorrect `spec` value on the `Node` object. The new codebase just ignores this issue entirely and I don't find that to be correct behaviour.